runtime/foucoco/Cargo.toml

```diff
@@ -25,6 +25,27 @@ smallvec = "1.9.0"
 # Local
 runtime-common = {path = "../common", default-features = false}
 
+# custom libraries from spacewalk
+currency = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+security = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+staking = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+oracle = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+stellar-relay = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+reward = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+fee = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+vault-registry = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+redeem = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+issue = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+nomination = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+replace = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+spacewalk-primitives = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+
+module-issue-rpc-runtime-api = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+module-oracle-rpc-runtime-api = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+module-redeem-rpc-runtime-api = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+module-replace-rpc-runtime-api = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
+module-vault-registry-rpc-runtime-api = { git = "https://github.com/pendulum-chain/spacewalk.git", default-features = false, branch = "update-orml-versions-to-0.9.36" }
```

Comment:
Since you removed the `.git` ending in the URLs elsewhere maybe we should remove it here too?

Reply:
i was going to put them (`.git`) back today. But I actually don't know which one is better.. having them or not?

Reply:
i removed the `.git`